app/src/test/java/com/example/marvelapp/presentation/mvp/utils/ConstantsUtils.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.marvelapp.presentation.mvp.utils
+
+object ConstantsUtils {
```

Comment:
if this class really necessary? If so, please put the constants names correctly

Reply:
Don thx